File: Palindrome.java
import java.util.*;

class Palindrome {
    public static void main(String args[]) {
        /* This program will tell the user wether the string they provided is a palindrome or not */
        String reversed = "";
        System.out.println("Hello! welcome to the Palindrometer. Please enter a string of characters to check:");
        Scanner input = new Scanner(System.in);
        String original = input.nextLine();
        input.close();
        int len = original.length();
        for (int i=len-1;i>=0;i--) {
            reversed += original.charAt(i);
        }
        if (original.equals(reversed)) {
            System.out.println("Your string is a palindrome!");
        } else {
            System.out.println("Your string is not a palindrome!");
        }
    }
}

File: 07 Basic JS/GeekOut6_Palindrome/js/palindrome.js
var str = prompt("Please enter a string");

function isPalindrome(str) {
  var palindrome = true;
  var j = str.length-1;
  for (var i=0; i<j+1; i++) {
    if (str.charAt(i) !== str.charAt(j)) {
      palindrome = false;
    }
    j--;
  }
  if (palindrome) {
    alert("This string is a palindrome");
  } else {
    alert("This string is not a palindrome");
  }
}

isPalindrome(str);